build.sbt

```diff
@@ -95,6 +95,79 @@ lazy val core = (project in file("core"))
     (compile in Compile) := ((compile in Compile) dependsOn createTargetClassesDir).value
   )
 
+lazy val oci = (project in file("delta-contribs-oci"))
+  .aggregate(core)
+  .dependsOn(core % "compile->compile;test->test")
+  .enablePlugins(GenJavadocPlugin, JavaUnidocPlugin, ScalaUnidocPlugin)
+  .settings (
+    name := "delta-contribs-oci",
+    commonSettings,
+    scalaStyleSettings,
+    // mimaSettings,
+    unidocSettings,
+    releaseSettings,
+    libraryDependencies ++= Seq(
```

Comment:
do we need to add these dependencies? I think this project automatically gets the dependencies from the core module